litecli/packages/special/dbcommands.py

```diff
@@ -56,6 +56,40 @@ def list_tables(cur, arg=None, arg_type=PARSED_QUERY, verbose=False):
     return [(None, tables, headers, status)]
 
 
+@special_command(
+    ".views",
+    "\\dv",
+    "List views.",
+    arg_type=PARSED_QUERY,
+    case_sensitive=True,
+    aliases=("\\dv",),
+)
+def list_views(cur, arg=None, arg_type=PARSED_QUERY, verbose=False):
+    if arg:
+        args = ("{0}%".format(arg),)
+        query = """
+            SELECT name FROM sqlite_master
+            WHERE type = 'view' AND name LIKE ? AND name NOT LIKE 'sqlite_%'
+            ORDER BY 1
+        """
+    else:
+        args = tuple()
+        query = """
+            SELECT name FROM sqlite_master
+            WHERE type = 'view' AND name NOT LIKE 'sqlite_%'
```

Comment:
Is there a specific reason to ignore views with prefix sqlite_? Do you happen to know when these(sqlite_xxx) views are created?

Reply:
I believe this query is a modified version of the tables_query in [sqlexecute.py](https://github.com/dbcli/litecli/blob/main/litecli/sqlexecute.py#L25-L29). SQLite will create temporary tables for query planning purposes such as `sqlite_stat*`. I don't think it is applicable to views.